packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.template.ts

```diff
@@ -0,0 +1,40 @@
+import { html, when } from "@microsoft/fast-element";
+import { BreadcrumbItem } from "./breadcrumb-item";
+import { Breadcrumb } from "../breadcrumb";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(BreadcrumbItem:class)} component.
+ * @public
+ */
+export const BreadcrumbItemTemplate = html<BreadcrumbItem>`
+    <div role="listitem" class="listitem" part="listitem">
+        ${startTemplate}
+        <slot>
+            <a
+                class="control"
+                part="control"
+                href="${x => x.href}"
+                aria-current="${x => (x.isCurrent ? "page" : void 0)}"
+            >
+                <span class="content" part="content">
+                    ${x => x.name}
+                </span>
```

Comment:
What's the purpose of having this span here?

Reply:
In this case, to style the underline of the anchor. Similar to how our fast-anchor is structured.